src/constants.ts

```diff
@@ -20,6 +20,7 @@ export enum Reducers {
   Aggregator = 'aggregator',
   App = 'app',
   Staking = 'staking',
+  Proposal = 'proposal',
```

Comment:
I think it should be a modal on a `proposals` page. Correct me if i'm wrong @meho99

Reply:
yes, that's correct. I think it would be a good idea to merge my sagas, slices from the branch `feature/subgraph-deployment` with yours @iamadilmughal

Reply:
Sure @meho99, will do the merging